tutorials.html

```diff
@@ -1,100 +1,40 @@
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/getting_started/training_and_evaluating.md">
-                  Training and Evaluating Models
-                </a>
-              </li>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/getting_started/configuration.md">
-                  Configuring Experiments
-                </a>
-              </li>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/getting_started/creating_a_model.md">
-                  Creating a Model
-                </a>
-              </li>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/getting_started/using_as_a_library_pt1.md">
-                  Using AllenNLP as a Library (Part 1) - Datasets and Models
-                </a>
-              </li>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/getting_started/using_as_a_library_pt2.md">
-                  Using AllenNLP as a Library (Part 2) - Predictions and Demos
-                </a>
-              </li>
-            </ul>
-            <h2>Deeper Dives</h2>
-            <ul>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/notebooks/data_pipeline.ipynb"</a>
-                  Datasets, Instances, and Fields
-                </a>
-              </li>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/notebooks/vocabulary.ipynb">
-                  Vocabularies
-                </a>
-              </li>
-            </ul>
-            <h2>How Tos</h2>
-            <ul>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/how_to/elmo.md"</a>
-                  Using ELMo
-                </a>
-              </li>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/how_to/laziness.md">
-                  How to Train a Model with Lazy Data
-                </a>
-              </li>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/how_to/visualizing_model_internals.md">
-                  Visualizing model internals in a live demo
-                </a>
-              </li>
-            </ul>
-          </div>
-        </div>
-      </div>
+<h1>Introduction</h1>
 
-      {% include footer.html %}
-    </div>
-    {% include svg-sprite.html %}
-    {% include scripts.html %}
-  </body>
-</html>
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla a egestas libero. Suspendisse eu ex a nibh euismod semper. Nullam sed semper nulla, eget accumsan odio. Pellentesque ut dapibus turpis.</p>
+
+<table class="annotated-code">
+  <tr>
+    <td class="code">
+
+{% highlight python %}
```

Comment:
@joelgrus this is all you need to do to get python highlighting. Due to the way browsers render whitespace of `<pre><code>...</code></pre>` structures, I would recommend:

1. Don't put any tabs in front of lines you don't want indented
2. Don't add carriage returns between the `{% highlight %}` open/close tags and the code they're wrapping

E.g.

```
{% highlight python %}
var sqlKeywords = secondary(-2);
if (ram) {
    supplyQueueHyperlink(subnet_gui, virus_program);
} else {
    trinitron.end_servlet /= python + dual_editor;
}
googleMacBaseband.username_integrated = alpha(methodNetbiosKey);
excelLossyToken += -2;
router -= serial;
{% endhighlight %}
```

Outside of these code blocks, whitespace is not an issue so you can have your html formatted as clean or as messy as you want.